### PR TITLE
MobileNet and COCO-SSD Tutorials

### PR DESCRIPTION
**Pull request by jpyepez**
Hi! I've added MobileNet and COCO-SSD tutorials and the image folder for document resources.

### CODE REVIEW

File: docs/tutorial_cocossd.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+
+# Tutorial: Object Localization with COCO-SSD
+
+Runway facilitates object detection and identification in images and video. 
+In this tutorial, we will use COCO-SSD to find and label the elements shown in a movie.
+
+### Requirements
+* Runway Beta
+* Docker (If you don't have Docker installed, please follow the
```
Comment (cvalenzuela) [inline docs/tutorial_cocossd.md]: Docker is not needed to run COCO-SSD and Mobilenet

File: docs/tutorial_mobilenet.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
+# Tutorial: Image Classification with MobileNet
+
+Runway enables fast and simple image classification using lightweight
+neural networks. This tutorial will show you how use the MobileNet
+model to identify the objects in an image folder.
+
+### Requirements
+* Runway Beta
+* Docker (If you don't have Docker installed, please follow the
```
Comment (cvalenzuela) [inline docs/tutorial_mobilenet.md]: Same here!

**Comment by cvalenzuela** [comment]
Yey! Thanks for this. We just need to remove the Docker reference, besides that, it should be good to go!

**Comment by jpyepez** [comment]
No worries, glad to help! Docker references should be gone. :)

**Comment by cvalenzuela** [comment]
Great, merging Thanks @jpyepez!

**Pull request merged**